iphone/Classes/TiExceptionHandler.m

```diff
@@ -56,16 +57,29 @@ - (void)reportScriptError:(TiScriptError *)scriptError
 
 - (void)showScriptError:(TiScriptError *)error
 {
-  [[TiApp app] showModalError:[error description]];
-  [[NSNotificationCenter defaultCenter] postNotificationName:kTiErrorNotification
-                                                      object:self
-                                                    userInfo:error.dictionaryValue];
+  NSArray<NSString *> *exceptionStackTrace = [NSThread callStackSymbols];
+
+  if (exceptionStackTrace == nil) {
+    [[TiApp app] showModalError:[error description]];
+  } else {
+    NSMutableArray<NSString *> *formattedStackTrace = [[[NSMutableArray alloc] init] autorelease];
+    NSUInteger exceptionStackTraceLength = [exceptionStackTrace count];
+
+    // re-size stack trace and format results. Starting at index = 4 to not include the script-error API's
+    for (NSInteger i = 4; i <= (exceptionStackTraceLength >= 20 ? 20 : exceptionStackTraceLength); i++) {
```

Comment:
@garymathews This is causing an array out of bound exception:
```js
var win = Ti.UI.createWindow({
    backgroundColor: '#fff',
    homeIndicatorAutoHidden : true
});
 
var scrollable = Ti.UI.createScrollableView({
});

win.add(scrollable);
win.open();

setTimeout(function () {
    scrollable.setViews([-1]);
}, 4000);
```
We should probably do `i < xxxxx`?